public/src/components/campaign/joined_campaigns.jsx

```diff
@@ -43,12 +51,15 @@ export default class JoinedCampaigns extends Component {
               />
             </Col>
             <Col xs={7} md={8} lg={9}>
-              <CurrentCampaign
-                history={history}
-                id={current_campaign}
-                defaultMsg={joined_campaigns.length > 0 ? 'Select a campaign' : 'Join a campaign'}
-                campaign={current_campaign}
-              />
+              { joined_campaigns.length !== 0 ?
+              (
+                <CurrentCampaign
+                  history={history}
+                  id={current_campaign}
+                  defaultMsg={joined_campaigns.length > 0 ? 'Select a campaign' : 'Join a campaign'}
```

Comment:
I guess now that there will always be a current_campaign set if a user has joined campaigns, then maybe the defaultMsg should just be 'Join a campaign'

Reply:
thanks!